src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -709,26 +709,106 @@ public FieldTypeClass(int id, string name, int typeId, bool isNotPrivate, FieldA
     }
     internal class ValueTypeClass
     {
-        public byte[] valueTypeBuffer;
-        public JArray valueTypeJson;
-        public JArray valueTypeJsonProps;
+        private MonoSDBHelper sdb;
+        public byte[] buffer;
+        public JArray json;
+        public JArray jsonProps;
         public int typeId;
-        public JArray valueTypeProxy;
-        public string valueTypeVarName;
-        public bool valueTypeAutoExpand;
+        public JArray proxy;
+        public string variableName;
+        public bool autoExpand;
```

Comment:
- Since we are changing these anyway, we can:

  - make the names `CamelCase`
  - make them properties
  - And keep only the needed members as public

```csharp
private JArray json;
private JArray jsonProps;
private JArray proxy;
private bool autoExpand;

public byte[] Buffer { get; }
public int TypeId { get; }
public int Id { get; }
```